external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/KafkaSpoutEmitTest.java

```diff
@@ -100,6 +103,8 @@ public void testNextTupleEmitsFailedMessagesEvenWhenMaxUncommittedOffsetsIsExcee
             when(consumerMock.poll(anyLong()))
                 .thenReturn(new ConsumerRecords<>(records));
 
+            OffsetAndMetadataStub.committed(consumerMock);
```

Comment:
I tried checking out the branch and running the test without this line. It seems to still pass. Maybe we should just get rid of it (and OffsetAndMetadataStub)?

Reply:
Done